src/components/sideNavbar/components/NavItem.tsx

```diff
@@ -0,0 +1,29 @@
+// WIP
```

Comment:
Remove dangling comment or still WIP?

Reply:
Still WIP but I'll remove the comment. I would like to start testing the package on Task Management. I will definitely revisit this to try and get it finished.